docs/D-Modularity/pointers.md

```diff
@@ -1,5 +1,9 @@
 ---
+id: pointers
```

Comment:
This style of frontmatter is great.  I'm going to file an issue to standardize on it.